wandb-scim/examples/user_example.py

```diff
@@ -130,14 +138,34 @@ def assign_team_role_to_user(user, user_id, team_name, role_name):
     try:
         # Assign team role to a user
         assign_team_role_response = user.assign_team_role(
-            user_id,
-            request_payload={"roleName": role_name, "teamName": team_name}
+            user_id, request_payload={"roleName": role_name, "teamName": team_name}
         )
         print(assign_team_role_response)
     except requests.exceptions.RequestException as e:
         print(f"Error occurred during API request: {str(e)}")
 
- 
+
+def assign_registry_role_to_user(user, user_id, registry_name, role_name):
+    """
+    Assigns a registry-level role to a user.
+
+    Args:
+        user (User): An instance of the User class.
+        user_id (str): The ID of the user to assign the registry role to.
```

Comment:
```suggestion
        user_id (str): The ID of the user.
```
Maybe I'm missing something, but what other user could it be?

Reply:
Yeah, again, I agree with this in general and would prefer to shorten it, but for the sake of maintaining + enforcing established patterns, I think it might be best (the lesser of two evils) to keep it as-is